File: 200_code_python/7.py
# Viết bằng PYTHON
# Tính diện tích hình chữ nhật: Nhập chiều dài, chiều rộng và in ra chu vi, diện tích của hình chữ nhật.

# Nhập chiều dài và chiều rộng
a = float(input("Nhap chieu dai a: "))
b = float(input("Nhap chieu rong b: "))

# Kiểm tra chiều dài và chiều rộng phải lớn hơn 0
if a > 0 and b > 0 :
    # Tính chu vi và diện tích hình chữ nhật
    C = a + b
    S = a*b
    # In kết quả
    print("Chu vi hinh chu nhat la: ", C)
    print("Dien tich hinh chu nhat la: ", S)
else:
    print("Chieu dai va chieu rong phai khac 0.")

File: 200_code_cpp/7.cpp
// Viết bằng C++
// Tính diện tích hình chữ nhật: Nhập chiều dài và chiều rộng và in ra chu vi, diện tích hình chữ nhật.

#include <iostream>
using namespace std;

int main () {
    // Khai báo biến
    float a, b;

    // Nhập chiều dài, chiều rộng từ bàn phím
    cout << "Nhap chieu dai a: ";
    cin >> a;
    cout << "Nhap chieu rong b: ";
    cin >> b;

    // Kiem tra điều kiện chiều dài và chiều rộng phải lớn hơn 0
    if (a>0 && b>0 ) {
        // Tinh chu vi hình chữ nhật
        float C = a + b;
        // Tinh dien tich hình chữ nhật
        float S = a*b;
        // In kết quả
        cout << " Chu vi cua hinh chu nhat la: " << C << endl;
        cout << " Dien tich cua hinh chu nhat la: " << S << endl;
    } else {
        cout << "Chieu dai và chieu rong phai lon hon 0." << endl;
    }

    return 0;
}